src/compiler/scala/tools/nsc/typechecker/RefChecks.scala

```diff
@@ -1733,7 +1733,7 @@ abstract class RefChecks extends Transform {
         // type bounds (bug #935), issues deprecation warnings for symbols used
         // inside annotations.
         applyRefchecksToAnnotations(tree)
-        var result: Tree = tree match {
+        val result: Tree = tree match {
```

Comment:
Renaming this variable  `result` to `result1` could make for a smaller diff.

Reply:
I went back and re-did the change with step-wise IDE refactorings starting with your suggestion and moving to the same end result as this PR to be sure I hadn't changed the meaning.